src/transformers/models/gpt_bigcode/modeling_gpt_bigcode.py

```diff
@@ -211,6 +231,8 @@ def forward(
         encoder_hidden_states: Optional[torch.Tensor] = None,
         encoder_attention_mask: Optional[torch.Tensor] = None,
         use_cache: Optional[bool] = False,
+        padding_mask: Optional[torch.LongTensor] = None,
+        encoder_padding_mask: Optional[torch.LongTensor] = None,
```

Comment:
I think we should wait again for the padding mask refactor and not pass padding mask! #26792